core/controller/src/main/resources/reference.conf

```diff
@@ -6,6 +6,7 @@ whisk {
     use-cluster-bootstrap: false
   }
   loadbalancer {
+    managed-fraction: 90%
```

Comment:
This means, that a deployment which adjusts the blackbox-fraction today will have overlapping pools in the future if the managed-fraction is not adjusted?

Reply:
After your proposal of keeping the default in `group_vars/all`, the issue doesn't exists anymore here.